CyberDisc-RichPresence/obj/Debug/CyberDisc-RichPresence.csproj.FileListAbsolute.txt

```diff
@@ -0,0 +1,12 @@
+C:\Users\natko\Source\Repos\cd-rich-presence\CyberDisc-RichPresence\obj\Debug\CyberDisc-RichPresence.csprojAssemblyReference.cache

```

Comment:
This all makes reference to local file locations on your computer.

Reply:
As long as you use the release build or build it with the DLLs within the project folder, it works